content/en/docs/actionners/list.md

```diff
@@ -197,4 +197,27 @@ Example:
   actionner: calico:networkpolicy
   parameters:
     order: 20
+```
+
+### `aws:lambda`
```

Comment:
we need to small section to explain it will require an IAM role, Instance Profile or else, wdyt?

Reply:
That will be common between all AWS integrations, do you think it makes sense to have a separate section explaining how to use AWS actions and mentioning it here?

Reply:
yep, we need a separate section somewhere and just a reference here